### PR TITLE
Columns block: Navigation help #4

### LINKED ISSUE #4: Columns block: Navigation help
Columns area can contain a trigger for navigation help overlay. 

image

### PR DESCRIPTION
**Pull request by atopper**
## Issue

Fix #4

## 🔗 Test URLs:

- Before: https://main--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1-top-draft
- After: https://col_help_3--sgws-creative--hlxsites.hlx.page/bloominbrands/p/1-top-draft

Original: https://sgcreative.southernglazers.com/bloominbrands/p/1

## 📝 Description:
Embed the help icon in the paragraph text.  On a click, open the slider help overlay with close button.  On click, close the overlay.

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -512,4 +508,67 @@ main .back-to-home span.icon {
       opacity: 1;
     }
   }
+
+  /* Make a small left/right rotation/wiggle 6 times, then wait half the animation time and do it again. */
```
Comment (atopper) [inline styles/styles.css:512]: I needed to have the help 'wiggle/rotate' for 5 seconds.  Be still for 5 seconds.  And then repeat.
It is possible I could have done 1 wiggle as a keyframe, and called it, but this was direct and easier.

File: styles/styles.css [hunk 2]
```diff
@@ -401,15 +397,15 @@ main .section .default-content-wrapper .border-0 {
 }
 
 main .section .outback .default-content-wrapper .border-0 {
-  filter: drop-shadow(#00000080 0 -4px 1px);
+  filter: drop-shadow(0 -4px 1px #00000080);
```
Comment (tmathern) [inline styles/styles.css:400]: Just curious - why? What does it change?

**Review by tmathern — APPROVED** [review]
LGTM once linting and lighthouse are happy too!

File: styles/styles.css (hunk 2 above)
Comment (atopper) [reply styles/styles.css:400]: The was a style warning.  The 'template' wants the color at the end.  Should have no different effect.
image

**Review by allentsliu — APPROVED** [review]
LGTM!

**Comment by atopper** [comment]
image

**Pull request merged**